main.js

```diff
@@ -71,18 +73,20 @@ function allNumbersClicked(selectedOrder, sortedNumbers) {
     return (selectedOrder.length === sortedNumbers.length);
 }
 
-document.querySelector('#start').addEventListener('click', () => {
+function playGame() {
     const startingScore = 0;
     const randomNumbers = generateNumbers();
+    timeLeft = totalTime;
 
-    document.querySelector('#splash').style.display = 'none';
+    document.querySelector('#splash_screen').style.display = 'none';
+    document.querySelector('#game_over').style.display = 'none';
     document.querySelector('#game').style.display = 'block';
     document.querySelector('#player_score').textContent = `Score: ${startingScore}`;
 
-    displayTimeLeft(totalTime);
+    displayTimeLeft(timeLeft);
```

Comment:
again, why?

Reply:
this makes no difference. change it, dont, i dont mind

Reply:
again, it just made it more semantic (for myself at least)